Contributing.md

```diff
@@ -0,0 +1,47 @@
+# How to Contribute
+
+## What to Contribute
+
+We're looking for contributors from a variety of backgrounds -- not just biologists and other scientists interested in reading and synthesizing the literature, but people with experience running collaborative literature reviews or managing large projects in GitHub have also been extremely helpful.
+
+To look for ideas of what is currently needed, check [here](https://github.com/greenelab/covid19-review/issues/34) or look through the open [Issues](https://github.com/greenelab/covid19-review/issues) until you find a topic that interests you.
+
+## First Step
+
+First, go [here](https://github.com/greenelab/covid19-review/issues/17) and tell us a bit about yourself per the questions at the top.
+
+## Suggesting a Paper
+
+One of the best things you can do is suggest papers for review (even if it's outside of your own field but you think it seems interesting).
+To suggest a paper, fill in as much of the [New Paper Template](https://github.com/greenelab/covid19-review/issues/new/choose) as you feel comfortable with.
+
+## Summarizing a Paper
+
+If you find a [suggested paper](https://github.com/greenelab/covid19-review/labels/New%20Paper) you feel comfortable evaluating, copy the questions from the template and fill them out as best you can.
+Ideally we'd like two people to look at each paper.
+It's OK if you can't evaluate everything.
+Let us know what expertise you think would complement yours.
+
+## How to Contribute Text
+
+1. First, go [here](https://github.com/greenelab/covid19-review/issues/17) and tell us a bit about yourself. 
+2. Then, you can look through current document [as a pdf](https://greenelab.github.io/covid19-review/) or at each section in the [content](https://github.com/greenelab/covid19-review/content) folder.
```

Comment:
```suggestion
2. Then, you can look through [current document](https://greenelab.github.io/covid19-review/) or at each section in the [content](https://github.com/greenelab/covid19-review/content) folder.
```

Reply:
not sure if you wanted to link the PDF - the HTML was linked - I think the HTML is the way to go here anyway